src/man/firejail-profile.txt

```diff
@@ -254,6 +254,15 @@ blacklist /usr/bin/gcc*
 blacklist ${PATH}/ifconfig
 .br
 blacklist ${HOME}/.ssh
+.br
+
+.br
+Blacklisted files are visible, but get a size of 0 bytes, permissions 400,
+ownership set to root:root, and reset timestamps and extended attributes.
```

Comment:
Ownership is influenced by `noroot` option.

(And technically they get 0:0 I guess 😝 .)

Reply:
hmm when i combine blacklist with noroot, I get `Access: (0400/-r--------)  Uid: (65534/  nobody)   Gid: (65534/  nobody)`

Reply:
Yes, because there is no mapping for root (*no*root ...).

Reply:
yes but note the uid and guid of 65534, not 0. I will update the Pr to clarify.